HelperFunctions.ps1

```diff
@@ -457,9 +458,19 @@ function GetTestToolkitApps {
                                 @(get-childitem -Path "C:\Applications\*.*" -recurse -filter $_)
                             }
                         }
-                        $apps += @(get-childitem -Path "C:\Applications\*.*" -recurse -filter "Microsoft_Tests-*.app") | Where-Object { $_ -notlike "*\Microsoft_Tests-TestLibraries.app" -and ($version.Major -ge 17 -or ($_ -notlike "*\Microsoft_Tests-Marketing.app")) -and $_ -notlike "*\Microsoft_Tests-SINGLESERVER.app" }
+                        $apps += @(get-childitem -Path "C:\Applications\*.*" -recurse) | ? { $_.name -like "Microsoft_Tests-*.app" -or $_.FullName -match '\\Test\\.*[ _]Test(?:s?| Automations).app' } | Where-Object { ($version.Major -ge 17 -or ($_ -notlike "*\Microsoft_Tests-Marketing.app")) }
```

Comment:
It almost looks like it's easier just to explicitly list what to include and/or exclude.

Reply:
> as pr description says this would have been simpler / nicer if msft had a consistent folder structure and naming pattern :/

That we definitely agree on. But this is also not something we can change overnight.

Both explicitly listing the apps to include/exclude and matching the name with a patter have advantages and disadvantages.
I am more leaning towards the former as:
- it's easier to get an overview which apps are included/excluded
- new apps are not being added by default, which might be a good thing as there'll be no side effects.

What do you think?